dockerfiles/header_sync/Dockerfile

```diff
@@ -37,6 +37,7 @@ COPY --from=builder /vulcanizedb/vulcanizedb .
 COPY --from=builder /vulcanizedb/dockerfiles/header_sync/startup_script.sh .
 COPY --from=builder /vulcanizedb/db/migrations/* db/migrations/
 COPY --from=builder /go/src/github.com/pressly/goose/cmd/goose/goose goose
+COPY --from=builder /vulcanizedb/dockerfiles/wait-for-it.sh .
```

Comment:
I rarely say this, but, worth adding a comment here and/or putting something in the docs? Thinking I'd be tempted to delete this if I didn't see that it was used by `docker-compose`